task/docker-build/0.1/docker-build.yaml

```diff
@@ -19,7 +19,7 @@ spec:
     description: Reference of the image docker will produce.
   - name: builder_image
     description: The location of the docker builder image.
-    default: docker.io/library/docker:stable@sha256:18ff92d3d31725b53fa6633d60bed323effb6d5d4588be7b547078d384e0d4bf #tag: stable
+    default: docker.io/library/docker:19
```

Comment:
Can we also pin the version of the image by tagging the image with the SHA, as task will not fail if the image version gets updated.